Composer/packages/lib/code-editor/src/LuEditor.tsx

```diff
@@ -169,6 +174,13 @@ const LuEditor: React.FC<LULSPEditorProps> = (props) => {
         webSocket,
         onConnection: (connection: MessageConnection) => {
           const languageClient = createLanguageClient(formatMessage('LU Language Client'), ['lu'], connection);
+          languageClient.onReady().then(() =>
```

Comment:
dont repeat yourself, put this logic in a function and reuse it in L213

Reply:
simplified this function